week-1/Homework/extra/2-piping.js

```diff
@@ -16,43 +16,51 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  let sum = a + b;
+  return Number(sum.toFixed(2));
 }
 
-function multiply() {
-
+function multiply(c, d) {
+  let result = c * d;
+  return result;
 }
 
-function format() {
-
+function format(f) {
+  return "£" + f;
```

Comment:
This would probably be a really good place to do rounding; something like:
```javascript
return `£${sum.toFixed(2)}`;
```
because we generally want to round money for display, but not during calculation!

Then you'd need to pass the expression for the amount of money as an argument to `format`, rather than always passing the empty string :)